apps/test-app/app/components/Textarea.tsx

```diff
@@ -0,0 +1,27 @@
+import { FC } from "react";
+import { useField } from "remix-validated-form";
+
+export type TextareaProps = {
+  name: string;
+  label: string;
+  value?: string;
+  "data-testid"?: string;
+};
+
+export const Textarea: FC<TextareaProps> = ({
+  name,
+  label,
+  value,
+  "data-testid": dataTestId,
+}) => {
+  const { error, getInputProps } = useField(name);
+  return (
+    <>
+      <label>{label}</label>
+      <textarea {...getInputProps()} data-testid={dataTestId}>
+        {value}
```

Comment:
I don't think we need to worry about `value` for this component. We can remove this prop entirely.